packages/cyberstorm/src/components/AntiResult/AntiResult.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from "react";
+import styles from "./AntiResult.module.css";
+import { classnames } from "../../utils/utils";
+
+export interface Props {
+  children?: ReactNode | ReactNode[];
+  className?: string;
+}
+
+/**
+ * Ease of use component set for adding helpful content in
+ * place where content is missing
+ */
+export function AntiResult(props: Props) {
+  const { children, className, ...forwardedProps } = props;
```

Comment:
`props` accepts nothing but `children` and `className` so `forwardedProps` is unnecessary.